core/themes/basis/template.php

```diff
@@ -38,6 +38,19 @@ function basis_preprocess_page(&$variables) {
     $variables['classes'][] = 'view-name-' . $view->name;
   }
 
+  // Get the installed version.
+  $update_version = config_get('basis.settings', 'css_update_version');
+  $update_preference = config_get('basis.settings', 'css_update');
```

Comment:
These should _probably_ use `theme_get_setting()`?

```suggestion
  // Allow sub-theme settings to take precedence by using theme_get_setting():
  $update_version = theme_get_setting('css_update_version');
  $update_preference = theme_get_setting('css_update');
```

That would make it so that if a sub-theme was active, it could provide the settings in its theme settings form and config file instead of pulling from Basis. This would better handle situations where a subtheme is enabled but Basis is not (so there is no Basis settings form).

Reply:
I tested this and I think `theme_get_setting()` is correct here. See https://github.com/backdrop/backdrop-issues/issues/4782#issuecomment-2552589660

Reply:
I pushed a commit to change this to use `theme_get_setting()`.